freeipa/src/main/java/com/sequenceiq/freeipa/service/diagnostics/DiagnosticsCollectionValidator.java

```diff
@@ -45,6 +48,13 @@ public void validate(BaseDiagnosticsCollectionRequest request, Telemetry telemet
         }
     }
 
+    private void validateStackStatus(Stack stack) {
+        if (stack.getStackStatus().getStatus().isFreeIpaUnreachableStatus()) {
+            throw new BadRequestException(String.format("Cannot run diagnostics on stack (crn: '%s') if the stack is in %s state",
```

Comment:
I think the crn is not useful for the enduser also do not call it stack but use CDP terms like Data Hub or Data Lake

this class seems to me a duplication.